index.rst

```diff
@@ -13,17 +13,188 @@
+Hence cherry-picking is just annoying with no particular benefit.
+
+Outputs
+-------
+
+**Before:** Notebook contributors need to remember to clear outputs before checking in new versions of the notebooks.
+
+**After:** Have this happen automatically (via pre-commit hook or similar), or at least raise a CI warning if there is output checked in.
+
+**Discussion:**
+
+Having the human remember to clear outputs before saving and checking in is error prone. Even if the notebook ends in clear_outputs(), it still implies it was run to the end before commit.
+Ideally something like https://github.com/srstevenson/nb-clean would be integrated in the development workflow.
+
+This may also be of use to other notebook repo maintainers.
+
+
+Write-Only
+----------
+
+**Before:** Notebooks are checked out write-only in nublado to avoid conflicts
+
+**After:** Tutorial-notebooks continue to be write-only (this has saved a lot of support headaches) but system-test notebooks can be read-write to allow for easier guided troubleshooting.
+
+Directories
+-----------
+
+**Before:** Mobu runs notebooks at the root level of repos [?]
+
+**After:** Indicate to mobu whether to run a subdirectory (through eg. a .run file)
+
+**Discussion:** It would be nice to have a top-level directory structure for multiple runable repos, particularly for the data curation notebooks.
+This is not terribly important.
+
+Summit
+------
+
+**Before:** Mobu does not run at the summit
+
+**After:** There are system-test notebooks that probe basic "telescope" functionality (eg communication with DDS)
+
+**Discussion:** These had better be passive, we don't want to move the telescope or anything.... We should check what if any protections there are for this, eg is there further authorisation required to perform certain tasks
+
+Timing
+------
+
+**Before:** We have no timing information related to whole-notebook runs or per-cell runs
+
+**After:** Notebook and/or cell execution time can be monitored for duration
+
+**Discussion:** This has been controversial in discussion with the reasonable argument that notebook execution relies on too many factors and excursions from the norm are not determinative. The other side of the argument is that metrics would indicate the statistical as-is user experience for execution times and can provide at least coarse statistics (if not for alerting, at least for inspection).
+
+Any metrics should arguably be dispatched to sasquatch for monitor dogfooding purposes
+
+
+Recommended
+-----------
+
+**Before:** There is no automated checking of future recommended candidates
+
+**After:** A candidate recommended starts being mobu'ed (at least on data-int) as soon as it is identified
+
+**Discussion:** This is easily achieved technically - question is how to indicate this. Another container tag?
```

Comment:
Or just manually merging a change to Phalanx, since I think this has to be a manual step either way.

Reply:
Yeah good point. I have no strong feeling as long as it's a documented step. I guess it depends whether we think different sites will have different recommended builds. Hmm.

Reply:
Telescope and Site will, but other than that, I'd like to avoid it.